doc/dev/coding-guidelines-and-review.rst

```diff
@@ -29,8 +29,40 @@ indentation can be set with::
 For Python, `PEP 8 <http://www.python.org/dev/peps/pep-0008/>`__ is our
 standard. The ```pep8``` and ```autopep8``` Makefile targets are helpful. 
 
-Code, scripts, and documentation must have its spelling checked. Vim users can
-run::
+Code, scripts, and documentation must have their spelling checked. 
+
+Python-based `codespell` is the preferred tool. `codespell` can be installed
```

Comment:
Now that I understand codespell better, I disagree with it being 'preferred' :-/